app/javascript/actionsPanel/initializeActionsPanelToggle.js

```diff
@@ -29,9 +29,11 @@ export function initializeActionsPanel(user, path) {
     const panelDocument = modContainer.contentDocument;
 
     if (panelDocument) {
-      panelDocument
-        .getElementsByClassName('close-actions-panel')[0]
-        .classList.remove('hidden');
+      const closePanel = panelDocument.getElementsByClassName(
```

Comment:
Locally the spec was occasionally failing due to this closePanel being undefined, so I've added an extra guard here too